#702 Move discontinued scripts to dedicated directory

MichaelStritt opened the pull request:
### Linked issue

Check out #702

jan-petr reviewed (changes requested):
Agree that this is a good thing to do - but an issue for discussion. 

The files are now 'hidden' in the outdated directory, which is good. But we shall perhaps add this directory to the path, because otherwise, ExploreASL won't find them there and the users neither. So moving them to this directory is equal to deleting them - which we want to do only later.

So I would indeed move them there. Make sure that a big warning is printed if these functions are executed. But I would still put this directory to the initialized paths for ExploreASL.

MichaelStritt commented:
@jan-petr: I fixed the paths. It was a bit weird that we already had the full Developments folder within the search path. I removed that and added the Outdated scripts in 
c957a48 .

The warnings are already quite vivid:

![image](https://user-images.githubusercontent.com/46593074/124721314-4c231780-df09-11eb-9895-a3d96b7815f1.png)

![image](https://user-images.githubusercontent.com/46593074/124721190-2bf35880-df09-11eb-8ab2-b4850e2c4ab7.png)

In the warnings we also mention that we will move the ConfigFiles to CustomScripts within v1.8.0. Do you want to do this within this PR as well?

jan-petr reviewed (approved):
xASL_qc_CheckValidityJSON.m seems to be used by ExploreQC - we can move that to functions. All the rest is good.
